pinctrl/digital_interrupts.go

```diff
@@ -28,7 +28,7 @@ type DigitalInterrupt struct {
 // NewDigitalInterrupt constructs a new digitalInterrupt from the config and pinMapping. If
 // oldInterrupt is not nil, all channels added to it are added to the new interrupt and removed
 // from the old one.
-func NewDigitalInterrupt(
+func (ctrl *Pinctrl) NewDigitalInterrupt(
```

Comment:
These are good calls. Is old interrupt useful here in the long term? We can remove it eventually.

Reply:
not sure, old interrupt isn't used as far as i can tell but if an implementer wants to implement some kind of reconfigure logic then its still there